src/components/CookieClicker.vue

```diff
@@ -0,0 +1,411 @@
+
+      <div class="upgrades-section">
+        <h2>Upgrades</h2>
+        <div class="upgrades-list">
+          <div 
+            v-for="(upgrade, index) in availableUpgrades" 
+            :key="'upgrade-' + index"
+            class="upgrade-card"
+            :class="{ 
+              'can-afford': cookies >= upgrade.cost,
+              'purchased': upgrade.owned > 0
+            }"
+            @click="purchaseUpgrade(index)"
+          >
+            <div class="upgrade-header">
+              <h3>{{ upgrade.name }}</h3>
+              <span class="owned-count" v-if="upgrade.owned > 0">×{{ upgrade.owned }}</span>
+            </div>
+            <div class="upgrade-stats">
+              <p class="cost">Cost: {{ formatNumber(upgrade.cost) }} cookies</p>
+              <p class="value">+{{ upgrade.value }} CPS</p>
+            </div>
+            <div class="upgrade-description">
+              <p>{{ upgrade.description }}</p>
+            </div>
+          </div>
+        </div>
+      </div>
+  
+
+      <transition name="slide-fade">
+        <div v-if="showAchievementNotification" class="achievement-notification">
+          <span class="badge">✓</span>
+          Achievement Unlocked: {{ achievementNotificationText }}
+        </div>
+      </transition>
+    </div>
+  </template>
+  
+  <script>
+  export default {
+    name: 'CookieClicker',
+    data() {
+      return {
+        cookies: 0,
+        totalClicks: 0,
+        cookiesPerSecond: 0,
+        availableUpgrades: [
+          {
+            name: "Cursor",
+            description: "Automatic cookie clicking",
+            cost: 25,
+            value: 0.1,
+            owned: 0
+          },
+          {
+            name: "Grandma",
+            description: "A nice grandma to bake more cookies",
+            cost: 100,
+            value: 0.5,
+            owned: 0
+          },
+          {
+            name: "Farm",
+            description: "Grows cookie plants",
+            cost: 1100,
+            value: 4,
+            owned: 0
+          }
+        ],
+        showAchievementNotification: false,
+        achievementNotificationText: "",
+        cursorImage: require('../assets/upgrades/cursor.png'),
+        grandmaImage: require('../assets/upgrades/grandma.png'),
+        farmImage: require('../assets/upgrades/farm.png'),
+        cookieImage: require('../assets/cookie.png'),
+        imageErrors: {},
+        debugMode: false
+      }
+    },
+    computed: {
+      formattedCookies() {
+        return this.cookies.toFixed(1).replace(/\B(?=(\d{3})+(?!\d))/g, ",");
+      }
+    },
+    mounted() {
+      this.loadGame();
+      this.startGameLoop();
+    },
+    beforeDestroy() {
+      clearInterval(this.gameLoop);
+    },
+    methods: {
+      increment() {
+        this.cookies += 1;
+        this.totalClicks++;
+        this.checkAchievements();
+      },
+      getUpgradeStyle(index) {
+    const positions = [
+      { left: '15%', top: '20%' },
```

Comment:
Not sure what you are trying to do here but there are usually better ways to position images then absolute values since these will not respond well to changes in screen size